CHANGELOG.md

```diff
@@ -15,7 +45,7 @@ Highlights from this release include a bug fixed in DDD, extended documentation
 - Calibration API simplifications (#1763) [@natestemen]
 - W state Benchmarking Circuit (#1723) [@purva-thakre]
 - Consider id as slack in ddd circuit mask (#1744) [@Aaron-Robertson]
-- Remaining Stuff related to Identity Insertion Scaling in Docs (#1759) [@purva-thakre]
+- Remaining Stuff related to Identity Insertion Scaling in Docs (#1759) [@natestemen]
```

Comment:
Not sure why this got changed, but this was definitely Purva's PR.

Reply:
That's strange! Thanks for alerting me.